cmd/helper/authHelp.go

```diff
@@ -91,15 +104,18 @@ func Auth(cmd *cobra.Command, cfgFile string, conf *Config) {
 		}
 	} else {
 		fmt.Println("Getting token...")
-		authenticator.SetAuthInfo(clientID, secret)
 		http.HandleFunc("/callback", completeAuth)
 		go http.ListenAndServe(":8888", nil)
 		url := authenticator.AuthURL(state)
 		fmt.Println("Please log in to Spotify by clicking the following link, or copying it to a web browser:", url)
 		//wait for auth to finish
 		client := <-ch
+		if client == nil {
+			fmt.Println("Client is not initialized")
+			return
```

Comment:
I noticed this a couple places, is the intention here to have the program exit with the `return`, with this being an error case? In that case, would you mind changing these to `os.Exit(1)`?